src/panels/lovelace/views/hui-view.ts

```diff
@@ -219,9 +232,13 @@ export class HUIView extends LitElement {
     const hass = this.hass!;
     const lovelace = this.lovelace!;
 
-    if (lovelace.editMode && !editCodeLoaded) {
-      editCodeLoaded = true;
-      import(/* webpackChunkName: "hui-view-editable" */ "./hui-view-editable");
+    if (lovelace.editMode && !this.editCodeLoaded) {
```

Comment:
Don't move `editCodeLoaded` to the class

Reply:
I am not sure how to let the View know that the code was loaded and to only try and load edit mode once that is done without this